src/OpenDiffix.Core/Aggregator.fs

```diff
@@ -17,7 +17,8 @@ let cloneHashSet (hashSet: HashSet<'T>) = HashSet<'T>(hashSet, hashSet.Comparer)
 let private invalidArgs (values: Value list) =
   failwith $"Invalid arguments for aggregator: {values}"
 
-let private hashAid (aidValue: Value) = Value.hash aidValue
+let private hashAid (aidValue: Value) =
+  aidValue |> Value.toString |> Hash.string
```

Comment:
how consistent is consistent we desire here? Off the top of my head, firstly, I think `reference` appends `L` to the integers and floating point formatting rules might be different.

I'm guessing we have no use-case yet for exact match of the hashes between the 2 implementations, so we don't care about those details, but just making sure.

Reply:
Actually, I'm confused, it appears that we're hashing memory bytes in `pg_diffix`, not string representations, or am I missing something?

Reply:
Good catch, we hash integers directly in `pg_diffix`.